cloudinit/net/netops/iproute2.py

```diff
@@ -0,0 +1,93 @@
+from typing import Optional
+
+import cloudinit.net.netops as netops
+from cloudinit import subp
+
+
+class Iproute2(netops.NetOps):
+    @staticmethod
+    def link_up(interface: str, family: Optional[str] = None):
+        subp.subp(
+            ["ip"]
+            + (["-family", family] if family else [])
+            + ["link", "set", "dev", interface, "up"]
+        )
```

Comment:
If we are manipulating network config, I think it may be ideal to capture=True on these commands to aid in debugging, What do you think @holmanb? Too noisy?

Reply:
Isn't this already [the default behavior](https://github.com/canonical/cloud-init/blob/e8ef58b9c07a851d62777c68c34d26b8088597ff/cloudinit/subp.py#L164)? I removed it because it looks like a noop.

Reply:
Dumb, yes thanks for the correction there. My fault for not looking at the function sig when I made this comment. Please disregard.